### PR TITLE
LS: fix HTTP timeouts

### PR DESCRIPTION
**Pull request by RhinosF1**
This has caused outages before.

### CODE REVIEW

**Review by Reception123 — APPROVED** [review]

File: LocalSettings.php [hunk 1]
```diff
@@ -960,7 +960,15 @@
 	'wgWatchlistExpiry' => [
 		'default' => false,
 	],
-
+	
+	// HTTP
+	'wgHTTPConnectTimeout' => [
+		'default' => 3,
+	],
+	'$wgHTTPTimeout' => [
```
Comment (Universal-Omega) [inline LocalSettings.php:968]: Drop the ``$``.

**Review by Universal-Omega — CHANGES_REQUESTED** [review]

File: LocalSettings.php (hunk 1 above)
Comment (dmehus) [reply LocalSettings.php]: This has been fixed with https://github.com/miraheze/mw-config/pull/3799, which should be merged `first` to @RhinosF1's `patch` branch, then his PR to the `master` branch can be merged

File: LocalSettings.php [hunk 2]
```diff
@@ -960,7 +960,15 @@
 	'wgWatchlistExpiry' => [
 		'default' => false,
 	],
-
+	
+	// HTTP
+	'wgHTTPConnectTimeout' => [
+		'default' => 3,
```
Comment (Universal-Omega) [inline LocalSettings.php:966]: Should this not be 3.0, since the default is 5.0?

Comment (Universal-Omega) [reply LocalSettings.php:966]: Coping IRC conversation:

``` Allowed values: | (float) ```

This should be a float (3.0)

**Review by Universal-Omega — APPROVED** [review]

**Pull request merged**